### ISSUE TITLE
1/27 english

### ISSUE THREAD

**Issue by remotelocal** [remotelocal/web#57]
関係詞　先行詞

**Comment by remotelocal** [comment]
先行詞とは「コンサートに来た人たち」なら人たち

**Comment by remotelocal** [comment]
人はwho、ものはwhichを使う。さるはもの扱い

a monkey ~~who~~ **which** can speak english

**Comment by remotelocal** [comment]
the magazine which Shoko read yesterday.

**Comment by remotelocal** [comment]
the man who Nao may know~~s~~.
助動詞の後ろは動詞の原型っす。

**Comment by remotelocal** [comment]
the woman who love**s** the country.

ふつうに三単現はいります。

**Comment by remotelocal** [comment]
このコンサートに来た人たちはSOLTのファンに違いない

**The people who came to this concert** must be funs of SOLT.

関係詞のところは名詞一個分としてのセットとして扱って文を作る

**Comment by remotelocal** [comment]
That is the monkey which can speak English.

**Comment by remotelocal** [comment]
Akira got the magazine which Naoko read yesterday.

I made friends with the man who Nao may know today.

**Comment by remotelocal** [comment]
My father ~~don't~~ know the blog which I read everyday.
My father **does't** know the blog which I read everyday.

**Comment by remotelocal** [comment]
P 149
1. I don't know the people who came to the party last night.
2. We saw the cat which named Hiroshi at the theater.
   The name of the cat which we saw at the theater was Hiroshi.
3. The man who want to see you is at the door.
   There is a man who wants to see you at the door.
4. The boots which Kyoko usual wears is not sent by him.
   The boots which Kyoko **often** wears **are not** **the ones which he sent**.
